### PR TITLE
Update Ubuntu documentation to include mono web-server

### PR DESCRIPTION
**Pull request by dsame**
# Description
Improvement

I documented Mono web server

## Check list
- [x] Related issue / work item is attached
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1 [hunk 1]
```diff
@@ -28,14 +28,30 @@ function Get-NginxVersion {
     }
 }
 
+function Get-Xsp4Version {
+    $name = "xsp4"
+    $port = 8084
+    $version = ($(cat /tmp/xsp)  | Select-Object -first 1).Split(' ')[1]
```
Comment (al-cheb) [inline images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1:34]: Parse version from apt/dpkg.

File: images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1 [hunk 2]
```diff
@@ -28,14 +28,30 @@ function Get-NginxVersion {
     }
 }
 
+function Get-Xsp4Version {
+    $name = "xsp4"
+    $port = 8084
```
Comment (al-cheb) [inline images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1:33]: Parse port from the config file - /etc/default/mono-xsp4

File: images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1 [hunk 3]
```diff
@@ -28,14 +28,30 @@ function Get-NginxVersion {
     }
 }
 
+function Get-Xsp4Version {
+    $name = "xsp4"
```
Comment (al-cheb) [inline images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1:32]: nitpick: `mono-xsp4`

File: images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1 [hunk 4]
```diff
@@ -28,14 +28,30 @@ function Get-NginxVersion {
...
+    $port = 8084
+    $version = ($(cat /tmp/xsp)  | Select-Object -first 1).Split(' ')[1]
+    $serviceStatus =  ((/etc/init.d/mono-xsp4 status) | Select-Object -index 2).Split(' ')[6]
```
Comment (al-cheb) [inline images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1:35]: Use systemctl to get status:
`systemctl show -p ActiveState --value mono-xsp4`

File: images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1 (hunk 1 above)
Comment (thebeanespinoza83) [reply images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1]: - [ ]

**Review by thebeanespinoza83 — APPROVED** [review]

**Review by thebeanespinoza83 — APPROVED** [review]

**Review by thebeanespinoza83 — APPROVED** [review]

**Review by thebeanespinoza83 — APPROVED** [review]

**Review by thebeanespinoza83 — APPROVED** [review]

File: images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1 (hunk 2 above)
Comment (thebeanespinoza83) [inline images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1:33]: @thebeanespinoza83

**Review by thebeanespinoza83 — APPROVED** [review]

File: images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1 (hunk 3 above)
Comment (thebeanespinoza83) [reply images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1:32]: @thebeanespinoza83

File: images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1 (hunk 2 above)
Comment (thebeanespinoza83) [inline images/linux/scripts/SoftwareReport/SoftwareReport.WebServers.psm1:33]: #2486

**Review by thebeanespinoza83 — APPROVED** [review]

**Review by thebeanespinoza83 — APPROVED** [review]

**Review by thebeanespinoza83 — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Pull request merged**